Gemfile.lock

```diff
@@ -552,7 +563,7 @@ DEPENDENCIES
   will_paginate
 
 RUBY VERSION
-   ruby 3.0.0p0
+   ruby 2.6.6p146
```

Comment:
@ajaystha42 this change should not be present. We have ruby 2.6.6 for legacy reasons

Reply:
I believe this occured due to my ruby version in my local machine. I will fix it